src/test/resources/edu/ie3/datamodel/io/source/csv/_grid/default/line_input.csv

```diff
@@ -1,3 +1,3 @@
 uuid,geo_position,id,length,node_a,node_b,olm_characteristic,operates_from,operates_until,operator,parallel_devices,type
-92ec3bcf-1777-4d38-af67-0bf7c9fa73c7,"{""type"":""LineString"",""coordinates"":[[7.411111,51.492528],[7.414116,51.484136]],""crs"":{""type"":""name"",""properties"":{""name"":""EPSG:4326""}}}",test_line_AtoB,0.003,4ca90220-74c2-4369-9afa-a18bf068840d,47d29df0-ba2d-4d23-8e75-c82229c5c758,olm:{(0.00,1.00)},2020-03-24T15:11:31Z[UTC],2020-03-25T15:11:31Z[UTC],f15105c4-a2de-4ab8-a621-4bc98e372d92,2,3bed3eb3-9790-4874-89b5-a5434d408088
-91ec3bcf-1777-4d38-af67-0bf7c9fa73c7,"{""type"":""LineString"",""coordinates"":[[7.411111,51.492528],[7.414116,51.484136]],""crs"":{""type"":""name"",""properties"":{""name"":""EPSG:4326""}}}",test_line_CtoD,0.003,bd837a25-58f3-44ac-aa90-c6b6e3cd91b2,6e0980e0-10f2-4e18-862b-eb2b7c90509b,olm:{(0.00,1.00)},2020-03-24T15:11:31Z[UTC],2020-03-25T15:11:31Z[UTC],f15105c4-a2de-4ab8-a621-4bc98e372d92,2,3bed3eb3-9790-4874-89b5-a5434d408088
\ No newline at end of file
+92ec3bcf-1777-4d38-af67-0bf7c9fa73c7,"{""type"":""LineString"",""coordinates"":[[7.411111,51.492528],[7.414116,51.484136]],""crs"":{""type"":""name"",""properties"":{""name"":""EPSG:4326""}}}",test_line_AtoB,0.003,4ca90220-74c2-4369-9afa-a18bf068840d,47d29df0-ba2d-4d23-8e75-c82229c5c758,olm:{(0.00,1.00)},2020-03-24T15:11:31+00:00,2020-03-25T15:11:31+00:00,f15105c4-a2de-4ab8-a621-4bc98e372d92,2,3bed3eb3-9790-4874-89b5-a5434d408088
+91ec3bcf-1777-4d38-af67-0bf7c9fa73c7,"{""type"":""LineString"",""coordinates"":[[7.411111,51.492528],[7.414116,51.484136]],""crs"":{""type"":""name"",""properties"":{""name"":""EPSG:4326""}}}",test_line_CtoD,0.003,bd837a25-58f3-44ac-aa90-c6b6e3cd91b2,6e0980e0-10f2-4e18-862b-eb2b7c90509b,olm:{(0.00,1.00)},2020-03-24T15:11:31+00:00,2020-03-25T15:11:31+00:00,f15105c4-a2de-4ab8-a621-4bc98e372d92,2,3bed3eb3-9790-4874-89b5-a5434d408088
```

Comment:
If somebody can explain why timestamps with Z are not working here i would be glad to get some notice...

2020-03-24T15:11:31Z,2020-03-25T15:11:31Z

Reply:
You mean in `CsvGraphicSourceTest`?

Reply:
If yes, I think it's specifically because of this line:

https://github.com/ie3-institute/PowerSystemDataModel/blob/06a2731b808e8fcf964f301e2947d6d739b5d914/src/main/java/edu/ie3/datamodel/io/source/RawGridSource.java#L515

It looks like this is just a random error because for some reason, the creation order of the two lines gets switched if you apply the changes you mentioned. And I think because the lines are built in parallel, this can happen. 

The result of the changed creation order is that the first exception contains node UUID `bd837a25-58f3-44ac-aa90-c6b6e3cd91b2` instead of `4ca90220-74c2-4369-9afa-a18bf068840d`, so the error message does not match anymore and the test fails.

https://github.com/ie3-institute/PowerSystemDataModel/blob/06a2731b808e8fcf964f301e2947d6d739b5d914/src/test/groovy/edu/ie3/datamodel/io/source/csv/CsvGraphicSourceTest.groovy#L69

Reply:
One solution would just be to remove the last part of the expected error message (` 4ca90220-74c2-4369-9afa-a18bf068840d was not found for entity`), so then the order of failures does not matter anymore.